layouts/default.vue

```diff
@@ -6,7 +6,7 @@
 
 <script lang="ts" setup>
     if (typeof window !== "undefined" && "serviceWorker" in navigator) {
```

Comment:
This code registers a service worker for the current web page if the window object is defined and the browser supports service workers ("serviceWorker" in navigator).